manifests/sysmgmt.yaml

```diff
@@ -90,11 +90,11 @@ spec:
     namespace: services
   - name: cfs-trust
     source: csm-algol60
-    version: 1.6.1
+    version: 1.6.2
     namespace: services
   - name: cfs-hwsync-agent
     source: csm-algol60
-    version: 1.9.0-beta.1
+    version: 1.8.1
```

Comment:
I need to move the change into the release branch, which is tagged as beta, and increment it. I will do so and resolve this comment once I have done that.

Reply:
Fixed.